addons/mail/static/src/new/core/messaging_service.js

```diff
@@ -375,6 +375,9 @@ export class Messaging {
                     }
                     break;
                 case "mail.record/insert":
+                    if (notif?.payload?.Thread?.rtcInvitingSession) {
```

Comment:
Looks like the new notification handler is not getting the notification sent there, even though the legacy code does:

https://github.com/odoo-dev/odoo/blob/master-discuss-refactoring-rtc-invitationCards2-tso/addons/mail/models/mail_channel_member.py#L232

Reply:
Whereas this notification is correctly received on both sides:

https://github.com/odoo-dev/odoo/blob/master-discuss-refactoring-rtc-invitationCards2-tso/addons/mail/models/mail_channel.py#L472